virttest/ovirt.py

```diff
@@ -352,7 +370,11 @@ def delete_from_export_domain(self, export_name):
         """
         vm = self.lookup_by_storagedomains(export_name)
         try:
-            vm.delete()
+            sds_service = self.connection.system_service().storage_domains_service()
+            export_sd = sds_service.list(search=export_name)[0]
+            logging.info('Remove VM %s from export storage' % self.name)
+            export_vms_service = sds_service.storage_domain_service(export_sd.id).vms_service()
+            export_vms_service.vm_service(vm.id).remove()
```

Comment:
those code are duplicated to previous, can we wrap them into one function instead?